File: net/ShopErp.Domain/DeliveryOut.cs
using System;

namespace ShopErp.Domain
{

    public class DeliveryOut
    {
        public long Id { get; set; }
        public long ShopId { get; set; }
        public string OrderId { get; set; }
        public PopType PopType { get; set; }
        public PopPayType PopPayType { get; set; }
        public string DeliveryCompany { get; set; }
        public string DeliveryNumber { get; set; }
        public string ReceiverAddress { get; set; }
        public float Weight { get; set; }
        public float ERPDeliveryMoney { get; set; }
        public float ERPGoodsMoney { get; set; }
        public float PopDeliveryMoney { get; set; }
        public float PopCodSevFee { get; set; }
        public float PopGoodsMoney { get; set; }
        public string GoodsInfo { get; set; }
        public string Operator { get; set; }
        public DateTime CreateTime { get; set; }
    }
}

File: android/app/src/main/java/bjc/shoperp/domain/DeliveryOut.java
package bjc.shoperp.domain;

import java.util.Date;

/**
 * Created by hth on 2018/2/25.
 */

public class DeliveryOut {
    public long Id;
    public long ShopId;
    public String OrderId ;
    public String DeliveryCompany ;
    public String DeliveryNumber ;
    public String ReceiverAddress ;
    public float Weight ;
    public float ERPDeliveryMoney ;
    public float ERPGoodsMoney ;
    public float PopDeliveryMoney ;
    public float PopCodSevFee ;
    public float PopGoodsMoney;
    public String GoodsInfo ;
    public String Operator ;
    public Date CreateTime ;

    public String CreateTimeStr;
}
